plugins/CoreAdminHome/lang/en.json

```diff
@@ -104,6 +104,10 @@
         "YouMayOptOutBis": "To make that choice, please click below to receive an opt-out cookie.",
         "OptingYouOut": "Opting you out, please wait...",
         "ProtocolNotDetectedCorrectly": "You are currently viewing Matomo over a secure SSL connection (using https), but Matomo could only detect a non secure connection on the server. ",
-        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s"
+        "ProtocolNotDetectedCorrectlySolution": "To make sure Matomo securely requests and serves your content over HTTPS, you may edit your %1$s file and either configure your proxy settings, or you may add the line %2$s below the %3$s section. %4$sLearn more%5$s",
+        "MissingTrackingCodeEmailSubject": "%s is still missing the Matomo tracking code",
+        "JsTrackingCodeMissingEmail1": "Just a quick note - we checked and your website, %s, does not seem to have the Matomo tracking code installed.",
```

Comment:
Maybe we could have it a bit more neutral as it might be a mobile app or so? It might not even be the tracking code but a mobile sdk etc. Maybe we could make it dependent on the type? Or maybe we can make the message more positive. Instead of saying "you haven't done this or that" could say "To start tracking and getting insights install the tracking code now."

Reply:
I'm not sure it's really an issue for other types of sites. There's no easy way (AFAIK) to start tracking in a mobile app (I mean it will always take real developer time, they can't just copy paste some code), so focusing on websites should be ok. But I will tweak the messages to take other types of measurables into account.